lib/commands/install.ts

```diff
@@ -0,0 +1,83 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+
+export class InstallCommand implements ICommand {
+	private _projectFilePath: string;
+	
+	constructor(private $fs: IFileSystem,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $options: IOptions,
+		private $injector: IInjector,
+		private $staticConfig: IStaticConfig) { }
+		
+	public enableHooks = false;
+		
+	public allowedParameters: ICommandParameter[] = [];
+	
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			let projectFilePath = this.getProjectFilePath(args[0]);
+			let projectData = this.getProjectData(projectFilePath).wait();
+			let projectName = projectData.id.split(".")[2];
+			
+			this.$injector.resolve("projectService").createProject(projectName).wait(); 
+			
+			this.$options.path = path.join(this.$options.path || path.resolve("."), projectName);
+			
+			this.$logger.info("Adding platforms...");
+			
+			let $platformsData = this.$injector.resolve("platformsData");
+			let $platformService = this.$injector.resolve("platformService");
+			_.each($platformsData.platformsNames, platform => {
+				let platformData = $platformsData.getPlatformData(platform);
+				let frameworkPackageData = projectData[platformData.frameworkPackageName];
+				if(frameworkPackageData && frameworkPackageData.version) {
+					$platformService.addPlatforms([`${platform}@${frameworkPackageData.version}`]).wait();
+				}
+			}); 
+			
+		}).future<void>()();
+	}
+	
+	public canExecute(args: string[]): IFuture<boolean> {
+		return (() => {
+			if(!args[0]) {
```

Comment:
shouldn't we check current directory for package.json instead of failing here? 
I prefer to write `$ tns install` instead of `$ tns install .`